music_player2/music_player2/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2022 Xiaomi Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+-include $(APPDIR)/Make.defs
+
+ifeq ($(CONFIG_LVX_USE_DEMO_VELA_AUDIO), y)
+PROGNAME = music_player2
+PRIORITY = 100
+STACKSIZE = 16384  # Reduce stack size from 40KB to 16KB
+MODULE = $(CONFIG_LVX_USE_DEMO_VELA_AUDIO)
```

Comment:
Here is VELA_AUDIO, but config is `LVX_USE_DEMO_MUSIC_PLAYER`
Need to be the same one